constructors.js

```diff
@@ -35,10 +35,24 @@ function expect(target) {
 // \___/\____/_/ /_/____/\__/_/   \__,_/\___/\__/\____/_/  /____/
 //
 // Only add code to *THIS* section!
+function Dog (obj){
+this.color = obj ? obj.color :true;
+this.status = "normal";
+this.hungry =obj && obj.hungry !== undefined ? obj.hungry: true;
+
+};
+
+function Human (obj){
+this.cool = obj ? obj.cool: false;
```

Comment:
This code all works perfectly fine, however the formatting is a bit weird to look at. Try to keep your formatting in check.